File: sync/rest/maps/update-map/update-map.5.x.rb
require 'twilio-ruby'

# Initialize the client
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Retreive the service
service = client.sync.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')

# Update a Map TTL, set to expires in 10 days
map = service.sync_maps('MyFirstList').update(ttl: 864000)

puts "SID: #{map.sid}, expires: #{map.date_expires}"

File: sync/rest/maps/update-map/update-map.5.x.js
// To set up environmental variables, see http://twil.io/secure
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const Twilio = require('twilio').Twilio;

const client = new Twilio(accountSid, authToken);
const service = client.sync.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX');

service.syncMaps("Players")
  .update({
    ttl: 864000 // expires in 10 days
  })
  .then(map => {
    console.log(map.dateExpires);
  })
  .catch(error => {
    console.log(error);
  });
